privacyidea/lib/tokens/webauthn.py

```diff
@@ -455,6 +458,19 @@ def __init__(self,
         self.icon_url = icon_url
         self.authenticator_selection_list = authenticator_selection_list
         self.location = bool(location)
+        self.exclude_credentials = []
+        if credential_ids:
+            for cred_id in credential_ids:
+                self.exclude_credentials.append({
+                    "id": cred_id,
+                    "type": "public-key",
+                    "transports": [
+                        "usb",
+                        "ble",
+                        "nfc",
+                        "internal"
```

Comment:
Maybe we should use the `DEFAULT_ALLOWED_TRANSPORTS` here:
https://github.com/privacyidea/privacyidea/blob/6a74ff77cebf429b47f5b2f67f3bedd06f020f1d/privacyidea/lib/tokens/webauthntoken.py#L417=

Reply:
`DEFAULT_ALLOWED_TRONSPORTS` needs to be imported from `webauthntoken.py` which in turn imports `webauthn.py` (circular imports or reordering the constants)
I will currently use `TRANSPORTS` from within the `webauthn.py` file itself.